funnel/views/venue.py

```diff
@@ -74,11 +74,8 @@ def update_venue_settings(self):
                         room_obj.bgcolor = room['color'].lstrip('#')
                         room_obj.seq = room['seq']
                         db.session.add(room_obj)
-        try:
-            db.session.commit()
-            return {'status': True}
-        except Exception as e:
-            return {'error': str(e)}, 400
+        db.session.commit()
+        return {'status': True}
```

Comment:
Don't know what exception was being raised here. Removing it till it throws an exception so that we can add a proper handler.